tests/regression_tests/photon_source/test.py

```diff
@@ -52,6 +54,8 @@ def _get_results(self):
             outstr += 'tally {}:\n'.format(t.id)
             outstr += 'sum = {:12.6E}\n'.format(t.sum[0, 0, 0])
             outstr += 'sum_sq = {:12.6E}\n'.format(t.sum_sq[0, 0, 0])
+            outstr += 'max_lost = {:12.6E}\n'.format(sp.n_max_lost_particles)
+            outstr += 'rel_max_lost = {:12.6E}\n'.format(sp.relative_max_lost_particles)
```

Comment:
Along the same lines, if these are removed from the statepoint, they shouldn't be in the test either. I think it's sufficient to have the unit test that confirms the proper values were put into place.